src/foam/core/stdlib.js

```diff
@@ -998,6 +998,23 @@ foam.LIB({
       }
 
       return date.toDateString().substring(4);
+    },
+    /** formats a date as MMM dd, YYYY
+     *  Use the timeFirst parameter to include a time string
+     *  "hh:mm:ss" at the start or end of the date string
+     **/
+    function formatDate(date, timeFirst) {
```

Comment:
We have DateTime, Time, and Date models. 
Perhaps each implements format() for itself, and with only DateTime using the timeFirst argument.

Reply:
updated datetime and date, but looks like time is already a formatted string
please re-review, thanks @jlhughes